### ISSUE TITLE
Regarding the issue of tracking RGB information

### ISSUE THREAD

**Issue by Jingranxia** [WangLongZJU/DeepAC#11]
Hello author, I would like to ask another question. If the stl file drawn from my item modeling file does not have color information on it, will there still be excellent tracking effects for the item with rich textures?

**Comment by WangLongZJU** [comment]
Yes, DeepAC uses the local region around projected contours to optimize the object pose. 
Whether your models have rich textures or not, we believe the local region can provide sufficient information to predict the boundary probability if there is some difference between the foreground and the background.
